dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoriesResource.java

```diff
@@ -91,6 +92,9 @@ public class CategoriesResource {
     private final PermissionAPI permissionAPI;
     private final CategoryHelper categoryHelper;
 
+    private static final String PAGINATION_PER_PAGE_HEADER_NAME = "X-Pagination-Per-Page";
```

Comment:
I believe the `PaginationUtil` class already has all the logic you just added in there. Please re-use the Util we already have. See the following:

- Its declaration: https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotcms/util/PaginationUtil.java#L46-L49
- How it's being used: https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java#L191

Reply:
PAGINATION_PER_PAGE_HEADER_NAME
PAGINATION_CURRENT_PAGE_HEADER_NAME
PAGINATION_TOTAL_ENTRIES_HEADER_NAME

All the above are private fields inside PaginationUtil and for the Children categories we have a different way to implement pagination therefore I exposed only the required fields for the FE

@jcastro-dotcms

Reply:
BTW I have made the above fields public inside PaginationUtil so that I could use them in CategoriesResource